com.github.Qv2ray.json

```diff
@@ -0,0 +1,91 @@
+{
+    "app-id": "com.github.Qv2ray",
```

Comment:
Just noticed this, but you don't really own this domain name. See https://docs.flatpak.org/en/latest/using-flatpak.html#identifier-triples & https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic
Something more like `com.github.lhy0403.Qv2ray` could work.

Reply:
ok